catalog.html

```diff
@@ -0,0 +1,322 @@
+            <input name="rating" id="C2" type="checkbox">
+            <label for="C1" class="label_rating"></label>
+            <input name="rating" id="C1" type="checkbox">
+            <button class="button" type="submit">Рейтинг<output name="sum"> 8,9 </output></button>
+
+          </li>
+
+          <li class="housing-cards">
+            <img src="" alt="Дорога-серпантин.">
+            <a class="housing-cards-link" href="housing.html">
+              <h3 class="housing-cards-title">GreenTree Inn</h3>
+            </a>
+            <a href="#" class="housing">Гостиница</a>
+            <a href="#" class="control">от 3000 ₽</a>
+
+            <form action="https://echo.htmlacademy.ru/" method="post"></form>
+            <button type="button" formtarget="housing-cards">Подробнее</button>
+            <button class="checkbox" type="button" formtarget="housing-list">В избранное</button>
+             <!---->
+            <form action="https://echo.htmlacademy.ru/" method="post"></form>
+            <label for="D5" class="label_rating"></label>
+            <input name="rating" id="D5" type="checkbox" checked>
+            <label for="D4" class="label_rating"></label>
+            <input name="rating" id="D4" type="checkbox" checked>
+            <label for="D3" class="label_rating"></label>
+            <input name="rating" id="D3" type="checkbox">
+            <label for="D2" class="label_rating"></label>
+            <input name="rating" id="D2" type="checkbox">
+            <label for="D1" class="label_rating"></label>
+            <input name="rating" id="D1" type="checkbox">
+            <button class="button" type="submit">Рейтинг<output name="sum"> 8,9 </output></button>
+          </li>
+        </ul>
+      </div>
+      <div class="addition">
+        <form action="https://echo.htmlacademy.ru/" method="post"></form>
+        <button class="pagination-show-more" type="button">Загрузить ещё</button>
+        <ul class="pagination">
+          <li class="pagination-item">
+            <a class="pagination-link" href="index.html">1</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link pagination-current">2</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">3</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">4</a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">
+              <span class="visually-hidden"></span>
+            </a>
+          </li>
+          <li class="pagination-item">
+            <a class="pagination-link" href="#">10</a>
+          </li>
+        </ul>
+      </div>
+      <div>
+        <span>Гостиниц на странице:</span>
+        <form action="https://echo.htmlacademy.ru/" method="post">
+          <select name="select-control" label="number">
```

Comment:
```suggestion
          <select name="select-control" aria-label="Сколько гостиниц показывать на странице:">
```

aria-label и label - это разные вещи